src/content/docs/dropins/checkout/checkout-functions.mdx

```diff
@@ -1,9 +1,563 @@
+
+- `fetchAddressFormFields`
+- `getCountries`
+- `getStoreConfig`
+
+### Usage
+
+See the following example for usage details:
+
+```typescript
+import {
+  fetchAddressFormFields,
+  getCountries,
+  getStoreConfig,
+} from '@/checkout/api';
+
+initializeCheckout();
+```
+
+### Returns
+
+The function returns a promise that resolves to an array containing the results of the three API calls.
+
+```ts
+Promise.all([getStoreConfig(), getCountries(), fetchAddressFormFields()]);
+```
+
+## isEmailAvailable
+
+Checks whether the specified email has already been used to create a customer account using the [`isEmailAvailable`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/is-email-available/) Adobe Commerce GraphQL API query.
+
+A value of `true` indicates that the email address is available and the customer can use the email address to create an account.
+
+### Returns
+
+A promise that resolves to an `EmailAvailability` object indicating whether the email is available.
+
+```ts
+type EmailAvailability = boolean;
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { isEmailAvailable } from '@/checkout/api/isEmailAvailable';
+
+isEmailAvailable('test@example.com');
+```
+
+## placeOrder
+
+Handles the process of placing an order in the checkout system using the [`placeOrder`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/place-order/) Adobe Commerce GraphQL API mutation.
+
+### Returns
+
+The transformed order data fetched from the API.
+
+```ts
+transformOrder
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { placeOrder } from '@/checkout/api/placeOrder';
+
+placeOrder('4L7E1H0muABuggqWxbzBCBWoeG0rNxDh');
+```
+
+## redirect
+
+A function that redirects the user to a URL. It takes a single parameter (`url`) of type `string`, which represents the URL to redirect to.
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { redirect } from '@/checkout/api/redirect';
+
+redirect('https://example.com');
+```
+
+## resetCheckout
+
+Resets the checkout process to its initial state by clearing the cart data.
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { state } from '@/checkout/lib/state';
+
+resetCheckout();
+```
+
+## resetCustomer
+
+Clears the current customer information and resets the customer state to its inital value in the checkout process.
+
+### Usage
+
+See the following example for usage details:
+
+```typescript
+import { state } from '@/checkout/lib/state';
+
+resetCustomer();
+```
+
+## setBillingAddress
+
+Sets the billing address for a specific cart using the [`setBillingAddressOnCart`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/set-billing-address/) Adobe Commerce GraphQL mutation.
+
+It takes the following parameters:
```

Comment:
We refactored the input parameters to keep compatibility with the mutation as follow:
`setBillingAddress({input: BillingAddressInput})`
...where 
```
BillingAddressInput {
  address?: AddressInput;
  customer_address_id?: number;
  same_as_shipping?: boolean;
  use_for_shipping?: boolean;
}
```
Then an example could be:
```
setBillingAddress({
  input: {
    address: {
      firstname: 'John',
      lastname: 'Doe',
      street: ['123 Main St', 'Apt 1'],
      city: 'New York',
      postcode: '10001',
      country_code: 'US',
      telephone: '555-555-5555',
    },
    same_as_shipping: false,
  }
});
```
I guess the mdx file is not up-to-date.